Jellyfin.Networking/Manager/NetworkManager.cs

```diff
@@ -759,45 +759,42 @@ public string GetBindAddress(IPAddress? source, out int? port)
             }
 
             // Get the first LAN interface address that's not excluded and not a loopback address.
+            // Get all available interfaces, prefer local interfaces
             var availableInterfaces = _interfaces.Where(x => !IPAddress.IsLoopback(x.Address))
                 .OrderByDescending(x => IsInLocalNetwork(x.Address))
                 .ThenBy(x => x.Index)
                 .ToList();
 
             if (availableInterfaces.Count > 0)
             {
+                // If no source address is given, use the preferred (first) interface
                 if (source is null)
                 {
                     result = NetworkExtensions.FormatIpString(availableInterfaces.First().Address);
```

Comment:
Should all occurrences of IP not be capitalized?

Reply:
I can enforce that on everything if you like. I only did it for the variables we discussed.

Reply:
I'd prefer if it was consistent tbh